lls/cache.c

```diff
@@ -299,9 +296,9 @@ lls_process_mod(struct lls_config *lls_conf, struct lls_mod_req *mod_req)
 unsigned int
 lls_process_reqs(struct lls_config *lls_conf)
 {
-	struct lls_request *reqs[LLS_CACHE_BURST_SIZE];
+	struct lls_request *reqs[lls_conf->mailbox_burst_size];
```

Comment:
Let's have a variable for the expression `lls_conf->mailbox_burst_size` as in the other patches.